frontend/public/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "SongBook",
+  "name": "The official Songbook of IT",
+  "icons": [
+    {
+      "src": "favicon.ico",
+      "sizes": "64x64 32x32 24x24 16x16",
+      "type": "image/x-icon"
+    }
+  ],
+  "start_url": "./index.html",
+  "display": "standalone",
+  "theme_color": "#000000",
```

Comment:
Not really the theme color right?

Reply:
I think i might just remove these. They arent used anywhere